src/utils/Pawapay.ts

```
@@ -0,0 +1,32 @@
+import { decimalRules } from "@/types/payments/pawapay";
```

ESLint

[error] Strings must use singlequote.

(style/quotes)